File: original_scripts/scala/bubble_sort/qwen3_1.7b.scala
def bubbleSort(arr: Array[Int]): Unit = {
  val n = arr.length
  for (i <- 1 to n - 1) {
    for (j <- 0 to n - 2 - i) {
      if (arr(j) > arr(j + 1)) {
        arr(j) = arr(j + 1)
        arr(j + 1) = arr(j)
      }
    }
  }
}

File: original_scripts/r/bubble_sort/qwen3_1.7b.r
BubbleSort
--------
function bubbleSort(arr) {
  let n = arr.length
  let swapped
  do {
    swapped = false
    for (let i = 0; i < n - 1; i++) {
      if (arr[i] > arr[i + 1]) {
        [arr[i], arr[i + 1]] = [arr[i + 1], arr[i]]
        swapped = true
      }
    }
    n--
  } while (swapped)
  return arr
}